### PR TITLE
Add basic circleci config file for running tests

### PR DESCRIPTION
**Pull request by wlach**

### CODE REVIEW

**Comment by whd** [comment]
I would base the circle config off of https://github.com/mozilla/telemetry-airflow/blob/master/circle.yml, otherwise you need to replicate the atmo build scripts in https://github.com/mozilla/telemetry-analysis-service/tree/master/bin. Also I'm seeing some duplicate commits, which is probably fallout from the repo move.

**Comment by wlach** [comment]
Yeah, I wasn't planning on using the build scripts, but the airflow scripts might be a better foundation to work off of regardless. Thanks!

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    # run tests using an upgraded version of docker-compose
+    - pip install --upgrade docker-compose
+  override:
+    - docker info
+    - docker --version
+    - docker build -t web .
+test:
+  override:
+    # run the tests using our test script
+    - docker-compose run web py.test
```
Comment (maurodoglio) [inline circle.yml]: Could you please add a `test` directive to the Makefile with this line in it?

**Comment by wlach** [comment]
Closing in favor of #10

**Pull request closed without merging**